website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,162 @@
+[ResourceQuota][k8s-resources-quotas] configuration in a namespace will cause all pods without 
+[Requests and limits][k8s-requests-limits] for `CPU` and `Memory` to be rejected.
+
+Because OLM-managed Operators do not run on Cluster infra nodes, they will consume paid CPU cores. 
+Users will naturally want to limit the maximum amount of resources these non-workload components can take up. 
+For this reason, if you are integrating with [OLM][olm], you should, **always ship your operator `Deployment` with resource requests amounts for CPU and memory**.
+
+The [ResourceQuota][k8s-resources-quotas] configuration also is applied to tenant workloads that are managed by your operator. Cluster administrators will typically set
+a [ResourceQuota][k8s-resources-quotas] for each tenant's namespace/project as part of the onBoarding. If your operator creates `Pods/Deployments` inside
+the tenant namespace which do not specify at least resource requests for `CPU` and `Memory`, those `Pods/Deployments` will fail to schedule and tenants
+will not be able to use your operator. Therefore, your operator should **always apply at least resource requests for CPU and
+memory** for `Pods/Deployments` that it creates as part of reconciliation. Ideally your operator also applies resource 
+limits to those `Pods/Deployments`. 
+
+#### General guidelines
+
+**Note that pretty much all controllers consume:**
+- `CPU`: largely based on the number of reconciliations they perform, which are generally related to event activity for resources they're watching.
+- `Memory`: largely based on the number of primary resources that exist (multiplied by some factor based on the number of operand resources they need to watch as a result) via informer caches.
+
+Operator authors cannot anticipate how they scale in usage in all customer environments, so setting one-size-fits-all 
+that fits well for all scenarios is not possible then, clusters administrators must be allowed to change the values 
+in order to respond to changes in their clusters’ needs, such as by resizing control plane nodes to provide more resources.
+
+**Following are some general recommendations to manage the resources:**
+
+- **MUST** declare resource requests for both CPU and memory.
+- **OPTIONALLY** setting the resources limit for Memory and CPU (Note that if your project is integrated with OLM then, admins would be able to customize it via the [subscription][olm-subscriptions]). Also, be aware that our project was built with Kubebuilder or SDK scaffold then for the `Operator/manager`(See `config/manager/manager.yaml`), then some values which allow you to get started are scaffolded by default, however, you ought to optimize based on the needs.
+- **Should** provide the mechanisms for [Monitoring compute & memory resource usage][k8s-metrics]. Kubebuilder and SDK provide some [metrics][kb-metrics] which can help you with. Note that cluster admins can use it to monitor and resize the Operator and its Operands.
```

Comment:
There is a caveat here. If the operator is installed by OLM and the bundle has a PodMonitor or a ServiceMonitor the complete InstallPlan will fail on a cluster, which does not have these CRD/the Prometheus operator installed. I am working on an [enhancement proposal](https://github.com/operator-framework/enhancements/pull/95) to avoid this situation as it is a significant drawback for operators provided by ISV

Reply:
This should be called out

Reply:
To solve the specific problem, "Operator is not installed because depends on/is using another Operator" that seems that the correct approach would be: 
a) Use [Operator Lifecycle Manager dependency resolution](https://docs.openshift.com/container-platform/4.6/operators/understanding/olm/olm-understanding-dependency-resolution.html)

Then, IMO operator-sdk helps integrate the projects with OLM and provide its helper. Also, we provide the metrics feature here. So, the best approach is we ensure that (as we do for other cases) that operator-SDK will scaffold what is required to allow the metrics feature to work on the cluster. 

is that makes sense? 
@jmrodri @fabiand @estroz  wdyt?

Reply:
it is out of the scope, so I tracked : https://github.com/operator-framework/operator-sdk/issues/5381